build.gradle

```diff
@@ -283,6 +283,8 @@ project('segmentstore:storage:impl') {
         compile group: 'org.rocksdb', name: 'rocksdbjni', version: rocksdbjniVersion
         testCompile project(':test:testcommon')
         testCompile project(path:':segmentstore:storage', configuration:'testRuntime')
+        testCompile group: 'org.slf4j', name: 'log4j-over-slf4j', version: slf4jApiVersion
```

Comment:
I don't think these two are needed.

Reply:
We can remove that, but I think this is the reason why there is no log printed when we run unit test on IDE yesterday. What do you think?